qiskit/circuit/library/arithmetic/piecewise_polynomial_pauli_rotations.py

```diff
@@ -101,9 +101,9 @@ def __init__(
             breakpoints: The breakpoints to define the piecewise-linear function.
                 Defaults to ``[0]``.
             coeffs: The coefficients of the polynomials for different segments of the
-            piecewise-linear function. ``coeffs[j][i]`` is the coefficient of the i-th power of x
-            for the j-th polynomial.
-                Defaults to linear: ``[[1]]``.
+                piecewise-linear function. ``coeffs[j][i]`` is the coefficient of the i-th power of x
+                for the j-th polynomial.
+                    Defaults to linear: ``[[1]]``.
```

Comment:
This indentation looks a bit strange, I believe this was wrong in the original and should be indented the same as the previous text 🤔